### DISCUSSION TITLE
关于功能的增加

### DISCUSSION THREAD

**Discussion by EMTEULA** [Arondight/Adachi-BOT#322] [Q&A] [answered]
能不能新增米游社的实时便筏，可以查询树脂信息的那种，我看到其他类似项目好像开发出来了，能不能整合一下，https://github.com/yaomeng0722/genshin_task-resin-expedition_alert

**Comment by Arondight** [comment, marked as answer]
我不会加入这个功能，如有需要可以自行合并这个 [PR](https://github.com/Arondight/Adachi-BOT/pull/301) 到你自己账户下的一个 fork

**Reply by EMTEULA** [reply]
感谢
